Bindings/Java/tests/TestEditProperties.java

```diff
@@ -7,6 +7,7 @@
   public static void main(String[] args) {
     try {
         Model model = new Model("gait10dof18musc_subject01.osim");
+        model.finalizeFromProperties();
```

Comment:
You are right, I didn't see the `initSystem()` call in the `OpenSimContext` construction call. I might have been over zealous in my changes. I can try backing it off.